### PR TITLE
Add max cape menu entries

### PR DESCRIPTION
**Pull request by vikke1234**
This patch adds "shorthands" for max cape teleports, an example can be seen below. 
image

### CODE REVIEW

**Comment by wa1id** [comment]
I would keep the naming consistent and change `Home` to `Tele to POH`

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 1]
```diff
@@ -435,6 +438,55 @@ default boolean teleportSubmenus()
...
+	{
+		WARRIORS_GUILD("Warrior's Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 0),
+		FISHING_GUILD("Fishing Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 1),
+		CRAFTING_GUILD("Crafting Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 2),
+		FARMING_GUILD("Farming Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 3),
+		OTTOS_GROTTO("Otto's Grotto", ComponentID.ADVENTURE_LOG_OPTIONS, 4),
+		RED_CHINS("Red Chinchompas", ComponentID.ADVENTURE_LOG_OPTIONS, 5, 1),
+		BLACK_CHINS("Black Chinchompas", ComponentID.ADVENTURE_LOG_OPTIONS, 5, 2),
+		HUNTER_GUILD("Hunter Guild", ComponentID.ADVENTURE_LOG_OPTIONS, 5, 3),
+		HOME("Home", ComponentID.ADVENTURE_LOG_OPTIONS, 6, 0),
```
Comment (wa1id) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java:451]: I would keep the naming consistent and change `Home` to `Tele to POH`

Comment (vikke1234) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java:451]: I suppose it depends on which naming you go by, as the menu says "Home" which is largely what I went with 
image

**Comment by vikke1234** [comment]
Moving this to custom menu swaps.

**Pull request closed without merging**